common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -338,6 +359,19 @@ div.problem {
       }
     }
 
+    &.partially-correct, &.ui-icon-check {
+      p.status {
+        display: inline-block;
+        width: 25px;
+        height: 20px;
+        background: url('../images/partially-correct-icon.png') center center no-repeat;
```

Comment:
Which looks like this: ![](https://github.com/edx/edx-platform/blob/master/common/static/images/partially-correct-icon.png) - ref https://github.com/edx/edx-platform/blob/master/common/static/images/partially-correct-icon.png

@marcotuts ?